Makefile

```
@@ -18,14 +19,21 @@ all: ${BUILD_DIR}
 ${BUILD_DIR}: CMakeLists.txt test/CMakeLists.txt
 	cmake -B${BUILD_DIR} .
 
+vcpkg:
+	git submodule update --init --recursive
```

It would be more explicit to phrase this terms of the toolchain file directly:

```suggestion
${TOOLCHAIN_FILE}:
	git submodule update --init --recursive
```

... and then have the `dev` and `dev3` targets depend on `${TOOLCHAIN_FILE}`.

---

done